composables/useScreenOrientation.ts

```diff
@@ -0,0 +1,17 @@
+import { ref, onMounted } from 'vue'
```

Comment:
I think we can actually move this to the `/utils` folder since it's just an esmodule which is great! 🙌  nice simplification

Reply:
@ermish should I keep it in its own folder or just in the utils straight away. Is it worth having a composable folder in case we make some later?